ui/imports/shared/views/profile/ShareProfileDialog.qml

```diff
@@ -131,11 +132,11 @@ StatusDialog {
                 background.color: "transparent"
                 background.border.color: Theme.palette.baseColor2
                 leftComponent: EmojiHash {
-                    publicKey: root.publicKey
+                    emojiHash: root.emojiHash
                     oneRow: true
                 }
                 rightComponent: CopyButton {
-                    textToCopy: Utils.getEmojiHashAsJson(root.publicKey).join("").toString()
+                    textToCopy: emojiHash.join("")
```

Comment:
Missing the `.toString()` part?

Reply:
there is no need too call `.toString()` because concatenation of strings is already a string